builder/dashboard_test.go

```diff
@@ -10,10 +10,10 @@ import (
 
 func TestDashboardWithError(t *testing.T) {
 	db := builder.NewDashboardBuilder(builder.AppId{
-		Project:     "mcoins",
-		Environment: "prod",
-		Family:      "marketing",
-		Application: "monetized-user-decider-revenue",
+		Project:     "myPrj",
+		Environment: "production",
+		Family:      "biz",
+		Application: "fancyBackend",
```

Comment:
we should use values like:
```
		Project:     "gosoline",
		Environment: "test",
		Family:      "montoring",
		Application: "dashboard"
```
so something more serious.. :)
I know, I'm not always the best example.. :D

Reply:
happily adjusting that :)